covid19model/py/etl.py

```diff
@@ -278,6 +274,27 @@ def fetchSocEc(dirPath):
     return p
 
 
+def gammaPar(mean, cv):
+    shape = 1 / cv ** 2
+    scale = mean / shape
+    return shape, scale
+
+
+def serialGammaDis(dateList, mean, cv):
+    print("\n~ Gamma Distribution TABLE ~")
+    shape, scale = gammaPar(mean, cv)
+    points = list(range(len(dateList)+1))
+    serial_gamma_density = gamma.pdf(points[1:], a=shape, scale=scale)
+    df = pd.DataFrame({"X": points[1:], "fit": serial_gamma_density})
+
+    print("--- saving Gamma Distribution table ---")
+
+    p = dirPath + "/SerialIntervalV2.csv"
```

Comment:
since we are generating it, let's delete the `covid19model/modelInput/SerialIntervalV2.csv` file from git